src/components/Bookmarks/List.spec.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { render, fireEvent } from '@testing-library/react-native'
+
+import Wrapper, { WrapperProps } from '../../lib/NavigatorContext'
+import defaultFolderData from '../../defaults/collections.json'
+import Screens from '../../lib/screens'
+
+import BookmarksList from './List'
+
+const bookmarkScreen = ( { children }:WrapperProps ) => (
```

Comment:
```suggestion
const bookmarkScreen = ( { children }: WrapperProps ) => (
```

unfortunately I'm not sure there's a lint rule for TS spacing

Reply:
unfortunately not Eslint or prettier don't pick it up.